taichi/ir/statements.h

```diff
@@ -839,17 +839,19 @@ class OffloadedStmt : public Stmt {
   using TaskType = OffloadedTaskType;
 
   TaskType task_type;
-  SNode *snode;
-  std::size_t begin_offset;
-  std::size_t end_offset;
-  bool const_begin, const_end;
-  int32 begin_value, end_value;
-  int step;
+  Arch device;
+  SNode *snode{nullptr};
+  std::size_t begin_offset{0};
+  std::size_t end_offset{0};
+  bool const_begin{false};
+  bool const_end{false};
+  int32 begin_value{0};
+  int32 end_value{0};
+  int step{0};
   int grid_dim{1};
   int block_dim{1};
-  bool reversed;
-  int num_cpu_threads;
-  Arch device;
+  bool reversed{false};
+  int num_cpu_threads{1};
```

Comment:
nit: sort `TI_STMT_DEF_FIELDS` at line 885 according to the order here?

Reply:
Done